charts/registry/templates/keycloak/database-credentials.yaml

```diff
@@ -22,7 +22,7 @@
 apiVersion: v1
```

Comment:
Regarding the postgres creds I would recommend to make it also possible, to use an already existing secret. Would it make sense to address this also in this PR, or should it be better in a separate PR?

Reply:
I will create a new Issue for this topic :) 
Thanks for the feedback.